attributes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dd-agent-builder": {
+      "omnibus-ruby_branch": "datadog-5.5.0"
+  },
+  "omnibus": {
+     "build_user_password": "DDBuildR0cks!"
```

Comment:
OK.  Most recent change requires the password in the environment (won't run if it's not there), so it's overriding all of the other coded defaults.
Password has been set in the environment in the gitlab runner.